src/backend/services/metrics.py

```diff
@@ -168,14 +174,18 @@ def get_agent(self, request: Request) -> Union[MetricsAgent, None]:
 
 
 async def report_metrics(data: MetricsData) -> None:
+    if not data:
+        logger.error("No metrics data to report")
```

Comment:
should we *throw* an exception here so we can trace back to which parts are sending empty data?

Reply:
as I understand, this code will log the error when it is thrown:
https://github.com/cohere-ai/cohere-toolkit/blob/d75f8083ed18fa7dac0b20d58f950aeaf053b44d/src/backend/services/metrics.py#L224-L225